CHANGELOG.md

```diff
@@ -1,5 +1,88 @@
+
+### Changed
+
+- TextInput now uses `forwardRef`, allowing it to be used directly by new APIs requiring a host component ([bbc5c35](https://github.com/facebook/react-native/commit/bbc5c35) by [@TheSavior](https://github.com/TheSavior))
+- Upgraded React to v16.11.0 ([1edce56](https://github.com/facebook/react-native/commit/1edce56) and [dff490d](https://github.com/facebook/react-native/commit/dff490d) by [@gaearon](https://github.com/gaearon) and [@TheSavior](https://github.com/TheSavior))
+- <TextInput> now uses React hooks internally ([dff490d](https://github.com/facebook/react-native/commit/dff490d) by [@TheSavior](https://github.com/TheSavior))
+- TouchableWithoutFeedback, TouchableOpacity, TouchableNativeFeedback, and TouchableHighlight are now classes without propTypes ([ebf7d75](https://github.com/facebook/react-native/commit/ebf7d75), [88ae24f](https://github.com/facebook/react-native/commit/88ae24f), [2185dd2](https://github.com/facebook/react-native/commit/2185dd2), and [7c01172](https://github.com/facebook/react-native/commit/7c01172) by [@yungsters](https://github.com/yungsters))
+
+#### Android specific
+
+- Change method signatures for ImageLoaderModule to accept double for requestId ([641e965](https://github.com/facebook/react-native/commit/641e965) by [@ejanzer](https://github.com/ejanzer))
+- Bump Gradle to 6.0.1 ([701e66b](https://github.com/facebook/react-native/commit/701e66b) by [@dulmandakh](https://github.com/dulmandakh))
+- Fix R8 warning ([bbb4e44](https://github.com/facebook/react-native/commit/bbb4e44) by [@dulmandakh](https://github.com/dulmandakh))
+- MainActivity launchMode is singleTask ([7a42596](https://github.com/facebook/react-native/commit/7a42596) by [@dulmandakh](https://github.com/dulmandakh))
+- Fix build failure due to duplicate libc++\_shared.so ([2fd5088](https://github.com/facebook/react-native/commit/2fd5088) by [@dulmandakh](https://github.com/dulmandakh))
+- Make DeviceInfo extend ReactContextBaseJavaModule ([5c64512](https://github.com/facebook/react-native/commit/5c64512) by [@RSNara](https://github.com/RSNara))
+- Avoid throwing exceptions when the host activity is not FragmentActivity ([7cfabf4](https://github.com/facebook/react-native/commit/7cfabf4))
+
+#### iOS specific
+
+- New internal image attribution support, but files importing RCTImageLoader.h must be converted to ObjC++ ([fdcdca4](https://github.com/facebook/react-native/commit/fdcdca4) by [@fkgozali](https://github.com/fkgozali))
+- Add another build flag for DevLoadingView ([3ceb959](https://github.com/facebook/react-native/commit/3ceb959) by [@mmmulani](https://github.com/mmmulani))
+
+### Removed
+
+- TouchableWithoutFeedback no longer exports Props; use React.ElementConfig instead ([7bcae81](https://github.com/facebook/react-native/commit/7bcae81) by [@yungsters](https://github.com/yungsters))
```

Comment:
This can probably be removed. It's only relevant to Flow types which (and even more specifically, only the way we use them at FB)